.scss-lint.yml

```diff
@@ -0,0 +1,4 @@
+linters:
+  Comment:
+    exclude:
+      - 'app/assets/stylesheets/application.css'
```

Comment:
The SCSS linter was complaining about the comments style in application.css. It prefers the use of `//` everywhere for comments, as opposed to `/* ... */`, but I think the asset pipeline needs the `*=` in there, so I thought the easiest thing to do for now is to disable the comment cop for this specific file (since we wouldn't be writing additional CSS directly in this file anyway).